packages/plugin-apex/yarn.lock

```diff
@@ -249,6 +249,14 @@
   dependencies:
     fancy-test "^1.4.3"
 
+"@salesforce/apex-node@0.2.8":
```

Comment:
we should remove this dependency, including this causes the plugin to use this version of `apex-node` rather than the local one while developing

Reply:
Good to know, thank you!